include/MainWindow.h

```diff
@@ -81,9 +82,25 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	static const int DEFAULT_AUTO_SAVE_INTERVAL = 2;
+	int m_autoSaveInterval;
+	const static int m_autoSaveShortTime = 10 * 1000; // 10s short loop
+
+	void autoSaveTimerReset( int msec = ConfigManager::inst()->
```

Comment:
Is it normal to init a default var this way?  It seems complicated and problematic.